low-level/sensors/impl/centos_7/systemd_watchdog.py

```diff
@@ -943,6 +945,7 @@ def _interface_added(self, object_path, interfaces_and_properties):
 
                 with self._drive_info_lock:
                     # Update drives
+                    self._drives[object_path] = {}

```

Comment:
this is only line fixing the issue ?

Reply:
Yes, `self._drives[object_path]` was not initialized before storing drive info
`self._drives[object_path][self.DRIVE_DBUS_INFO] = interfaces_and_properties['org.freedesktop.UDisks2.Drive']`

Reply:
ok